Backport of fix from xscreensaver-5.07 for 
6764033 xscreensaver can SEGV running interference hack

--- hacks/interference.c	Thu Mar 16 15:22:40 2006
+++ hacks/interference.c	Fri Jul 25 10:58:35 2008
@@ -383,7 +383,7 @@
 	dx = i*g + g/2 - c->source[k].x;
 	dy = j*g + g/2 - c->source[k].y;
 	dist = sqrt(dx*dx + dy*dy); /* what's the performance penalty here? */
-	result += (dist > c->radius ? 0 : c->wave_height[dist]);
+	result += (dist >= c->radius ? 0 : c->wave_height[dist]);
       }
       result %= c->colors;
 
